test/config_tests/config_v1_test.py

```diff
@@ -267,6 +267,23 @@ def test_config_cover(self):
                 travel_time_up=60,
             ),
         )
+        
+    def test_config_cover_device_class(self):
+        """Test reading cover with device_class from config file."""
+        self.assertEqual(
+            TestConfig.xknx.devices["Livingroom.Shutter_3"],
+            Cover(
+                TestConfig.xknx,
+                "Livingroom.Shutter_3",
+                group_address_long="1/4/5",
+                group_address_short="1/4/6",
+                group_address_position_state="1/4/7",
+                group_address_position="1/4/8",
```

Comment:
the tests break because the group addresses are wrong.
Please run tests locally before pushing to github - CI is taking forever to start lately...

Reply:
Hi @farmio , I'm not a professional software guy, but I'm willing to learn. Is there any guide how to run the tests locally? 
 I have understood that the group addresses have to be as follows:
```
group_address_long="1/4/9",
group_address_short="1/4/10",
group_address_position_state="1/4/11",
```

Reply:
Thats no problem, neither am I 😉

From your xknx directory:
- Install the dependencies, if you haven't yet with
```shell
pip3 install -r requirements/testing.txt
```
- run pytest for unit tests
```shell
pytest
# or just one specific test
pytest test/devices_tests/cover_test.py
```
- run tox for the whole test suite with unittests, linting, sorting and type checks run
```shell
tox
```
You can always join the discord server if you need any help - see eg. xknx.io for a link.